modular_sand/code/datums/interactions/interaction_datums/lewd/oral.dm

```diff
@@ -122,4 +124,5 @@
 									'modular_sand/sound/interactions/bj10.ogg',
 									'modular_sand/sound/interactions/bj11.ogg'), 50, 1, -1)
 	user.visible_message("<span class='lewd'><b>\The [user]</b> [message]</span>", ignored_mobs = user.get_unconsenting())
-	partner.handle_post_sex(lust_increase, CUM_TARGET_MOUTH, user)
+	if(partner.can_penetrating_genital_cum())
+		partner.handle_post_sex(lust_increase, CUM_TARGET_MOUTH, user)
```

Comment:
This interaction here also uses this both for the penis and vagina. This would make oral unable to make vaginas cum unless they're futas/have strapons